internal/driver/commands.go

```diff
@@ -129,115 +129,115 @@ var pprofCommands = commands{
 // reported generated by pprof.
 var pprofVariables = variables{
 	// Filename for file-based output formats, stdout by default.
-	"output": &variable{stringKind, "", "", helpText("Output filename for file-based outputs")},
+	"output": &variable{stringKind, []string{""}, "", helpText("Output filename for file-based outputs")},
```

Comment:
Good point. We also match non-numeric tags against "key:value" string which is used to match a specific tag. E.g. @rakyll had a [post](https://rakyll.org/profiler-labels) on this. I think the AND semantics in that matching are motivated by ability to do something like "-tagfocus foo:val1,bar:val2".

After a bit of back and forth thinking, maybe the simplest plan for now is to basically keep the current behavior but allow prefixing the -tagfocus value with "sometag=" prefix which will limit the tag focus filter to only take that single tag into account. This isn't as flexible as matching by different tags could be, but is a good start and should be a smaller change.

As an aside, I am not a fan of using the equal sign here since while it is possible to use a space as the flag value separator on the command line (e.g. "-tagfocus mytag=128kb:512kb"), the interactive pprof shell only takes equal sign as the separator for options and so user has to type something like "tagfocus=mytag=128kb:512kb" which is a bit ugly. @ahh in #178 proposed using "tagfocus=/mytag/128kb:512kb" syntax but I wanted to restrict the tag filtering to exact matches for now. Any thoughts on this are welcome. My take is double equal sign is not that bad.